background/get-popups.js

```diff
@@ -0,0 +1,34 @@
+const POPUP_PREFIX = chrome.runtime.getURL("popups");
+
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+  if (!request?.requestPopupInfo) return;
+  // For some reason non-popups managed to request popup info?
```

Comment:
Why

Reply:
he obviously doesn't know

Reply:
@apple502j Would be nice to know what's going on here